app/views/admin/friends/_basic_form_fields.html.erb

```diff
@@ -46,6 +46,13 @@
   </div>
 </div>
 
+<div class='row form-group'>
+  <%= f.label :eoir_case_status, class: 'col-md-2 control-label' %>
```

Comment:
Could we add the same text for this label that you have in app/views/friends/_info.html.erb ?
Here's an example where we specify the text of a form label: https://github.com/CZagrobelny/new_sanctuary_asylum/blob/ca0f8be97f15bcd34a0598744c6cc02b9311daf6/app/views/admin/friends/_basic_form_fields.html.erb#L144

Reply:
Comitted!